### PR TITLE
app-leechcraft/liblaretz: bump version

### PR DESCRIPTION
**Pull request by 0xd34df00d**
Bug: https://bugs.gentoo.org/628516
Signed-off-by: Georg Rudoy <0xd34df00d@gmail.com>

### CODE REVIEW

File: app-leechcraft/liblaretz/liblaretz-0.1.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2017 Gentoo Foundation
```
Comment (juippis) [inline app-leechcraft/liblaretz/liblaretz-0.1.2.ebuild]: Use repoman to commit to fix this copyright header automatically. This is the old form.

File: app-leechcraft/liblaretz/liblaretz-0.1.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (juippis) [inline app-leechcraft/liblaretz/liblaretz-0.1.2.ebuild]: Use 7, should work.

File: app-leechcraft/liblaretz/liblaretz-0.1.2.ebuild [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+DESCRIPTION="Shared library to be used by the Laretz sync server and its clients"
+HOMEPAGE="https://leechcraft.org"
+SRC_URI="https://github.com/0xd34df00d/${PN#lib}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+inherit cmake-utils
+
+LICENSE="Boost-1.0"
+SLOT="0"
+KEYWORDS="amd64 x86"
```
Comment (juippis) [inline app-leechcraft/liblaretz/liblaretz-0.1.2.ebuild]: You can't commit straight to stable, KEYWORDS need to be reset.

File: app-leechcraft/liblaretz/liblaretz-0.1.2.ebuild [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+IUSE=""
+
+DEPEND="dev-libs/boost"
```
Comment (juippis) [inline app-leechcraft/liblaretz/liblaretz-0.1.2.ebuild]: Verify whether this gets linked, ie, is subslot binding necessary here? `:=`

Comment (0xd34df00d) [reply app-leechcraft/liblaretz/liblaretz-0.1.2.ebuild]: Awesome, thanks for catching this! Indeed it does!

**Pull request closed without merging**